### PR TITLE
Add Aegis Authenticator icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/8385

**Popularity metric:**

It has ~8.2k stars on GitHub. See https://github.com/beemdevelopment/Aegis

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Resized, unifed paths, optimized.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -314,6 +314,11 @@
             "source": "https://www.adyen.com/press-and-media/presskit",
             "guidelines": "https://www.adyen.com/press-and-media/presskit"
         },
+        {
+            "title": "Aegis Authenticator",
+            "hex": "005E9D",
+            "source": "https://alexbakker.me/u/pn8frf828n.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:320]: How is the source at all related to the project?

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (CodeShakingSheep) [reply _data/simple-icons.json:320]: > There's now official vector by [Aegis Authenticator dev](https://github.com/alexbakker): https://alexbakker.me/u/pn8frf828n.svg

From the related issue: Alex Bakker is one of the developers of Aegis which is the only source for the Aegis svg. Unfortunately, the vector is neither hosted in their GitHub repo nor website.

Comment (michaelschattgen) [reply _data/simple-icons.json:320]: I am one of the two developers of Aegis and I can confirm that's our official vector image for Aegis Authenticator. If it helps we can host this image on our website.

Comment (uncenter) [reply _data/simple-icons.json:320]: > I am one of the two developers of Aegis and I can confirm that's our official vector image for Aegis Authenticator. If it helps we can host this image on our website.

I think it is fine but really would be nice if it was a more official URL, the obfuscated filename of `/u/pn8frf828n.svg` certainly doesn't do it any favors 😆

Comment (michaelschattgen) [reply _data/simple-icons.json:320]: @uncenter I agree, you're right! Just uploaded our vector image to our website :relaxed: thanks!

https://getaegis.app/aegis.svg

Comment (uncenter) [inline _data/simple-icons.json:320]:
```suggestion
            "source": "https://getaegis.app/aegis.svg"
```
Thank you again @michaelschattgen!

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by uncenter — APPROVED** [review]

**Pull request merged**